src/fe-text/irssi.c

```diff
@@ -313,6 +316,7 @@ int main(int argc, char **argv)
 	g_log_set_always_fatal(loglev);
 	textui_finish_init();
 	main_loop = g_main_new(TRUE);
+	dirty_check(); /* draw the screen prior to the first wait poll */
```

Comment:
I was considering that also but since we do dirty_check in the while loop it seems more appropriate here. another idea is to simply move the dirty_check before the poll inside the loop buut I didn't  want to evaluate the implications of that...?